### ISSUE TITLE
The lost quest item missing 

### ISSUE THREAD

**Issue by Enoxium** [Wynncraft/Issues#7455]
One of the letters for the quest was deleted when i tried to throw it out on accident and wasnt returned since my inventory was full /fixquest wont work please help Ign-Mawile_

**Comment by thecoolkogut4** [comment]
did you have an empty slot in your inventory when you type /fix quests? give all the letters that you currently have, then after they are all gone try /fixquests

**Comment by Pyromanical** [comment]
@thecoolkogut4 I don't think that works, the letters are Untradeables not Quest Items last time I checked.  There have been multiple issues about this on here and the forums.  Pretz said he was looking into this on one of the forums threads about this, but from what I've seen the only thing that can be done is get a new class up to that level to get another letter.  

This might have been fixed by now and what you said might work, it just depends on if they are untradeable or quest items.
